### ISSUE TITLE
Intel HD 4400 iGPU 

### ISSUE THREAD

**Issue by tapanmazumder** [dortania/bugtracker#39]
Please stop and read this carefully. Otherwise your issue will be left **unanswered** and **closed**.

* Only the latest released versions of OpenCore, Lilu, and other KEXTs are supported.
* Bug reports involving Chameleon, Clover, Ozmozis or alike are not accepted.
* Include your `EFI` directory with OpenCore.
* Include OpenCore DEBUG log (refer to `Target` in [Configuration.pdf](https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Configuration.pdf) or here: [OpenCore Debugging](https://dortania.github.io/OpenCore-Desktop-Guide/troubleshooting/debug.html)).
* Include Lilu DEBUG log (refer to `-liludbgall liludump=60` boot arguments in Lilu's [README](https://github.com/acidanthera/Lilu/blob/master/README.md)).

When to use [Acidanthera's issue tracker](https://github.com/acidanthera/bugtracker):

* You have new resources or patches to add but cannot make a pull request
* You have kernel panics, crashes, hangs and believe it is not a configuration issue
* You want to discuss technical or legal stuff

_If you have a kernel panic, please ensure that you have a DEBUG version of the extension and you have `-v keepsyms=1 debug=0x100` boot arguments added. On 10.13 or higher to avoid kext names scrolling over the panic log you should also set `PanicNoKextDump=YES` in OpenCore. Use `ApplePanic=YES` to write a kernel panic log to file._

**Comment by tapanmazumder** [comment]
I have Intel Core i3 4130 3.4 Ghz CPU (Haswell). I created Opencore 0.5.9 Mojave 10.14.6 from given instruction on https://dortania.github.io/OpenCore-Desktop-Guide/ it is listed that for intel HD 4400 we need to add the following poperties:
AAPL,ig-platform-id: 0300220D - this is used when the Desktop Haswell iGPU is used to drive a display

I set the attributes as per instruction only, but when I tried to boot it from Opencore USB it was stuck in apple logo. I have change my GPU Memory setting in BIOS and allocate 1024 to GPU, still it is stuck in apple logo with half loading.
